packages/service/src/commands/extract-messages.ts

```diff
@@ -8,9 +8,9 @@ import { packageRoot } from '../utils/path';
   description: 'Extract i18n default messages from .vue files.',
 })
 export class ExtractMessages implements ICommandHandler {
-  public async run(args: string[], silent: boolean) {
+  public async run(args: string[], options: IRunOptions) {
     try {
-      await runProcess('node', [packageRoot('dist/scripts/extract-i18n-messages.js')], { silent });
+      await runProcess('node', [packageRoot('dist/scripts/extract-i18n-messages.js')], { silent: options.silent });
```

Comment:
does it make sense to pass all the `options` object?  the above as well.

Reply:
maybe I could extend the runProcess interface with the IRunOptions interface, then it would be possible